src/query/ast/tests/it/testdata/statement-error.txt

```diff
@@ -531,7 +551,16 @@ error:
   --> SQL:1:37
   |
 1 | copy into t1 from "" FILE_FORMAT = (
-  |                                     ^ expected `TYPE`, `COMPRESSION`, `FORMAT_NAME`, `RECORD_DELIMITER`, `FIELD_DELIMITER`, `QUOTE`, `NAN_DISPLAY`, `NULL_DISPLAY`, `ESCAPE`, `ROW_TAG`, `SKIP_HEADER`, `NON_DISPLAY`, or `)`
+  | ----                                ^ expected `)`, `TYPE`, `COMPRESSION`, `FORMAT_NAME`, `RECORD_DELIMITER`, `FIELD_DELIMITER`, `QUOTE`, `NAN_DISPLAY`, `NULL_DISPLAY`, `ESCAPE`, `ROW_TAG`, `SKIP_HEADER`, or `NON_DISPLAY`
+  | |                                   
+  | while parsing `COPY
+                INTO { internalStage | externalStage | externalLocation | [<database_name>.]<table_name> }
+                FROM { internalStage | externalStage | externalLocation | [<database_name>.]<table_name> | ( <query> ) }
+                [ FILE_FORMAT = ( { TYPE = { CSV | JSON | PARQUET } [ formatTypeOptions ] } ) ]
+                [ FILES = ( '<file_name>' [ , '<file_name>' ] [ , ... ] ) ]
+                [ PATTERN = '<regex_pattern>' ]
+                [ VALIDATION_MODE = RETURN_ROWS ]
+                [ copyOptions ]`
```

Comment:
now will show example for `copy into statement`

Reply:
Great!

Reply:
@youngsofun  but the example seems outdated. the [document](https://databend.rs/doc/sql-commands/dml/dml-copy-into-table) is also outdated.

Reply:
The doc updated, remove the `NDJSON`.